### PR TITLE
Added test cases

### PR DESCRIPTION
**Pull request by uzaxirr-fam**
Screenshot 2023-04-03 at 10 44 50 AM

### CODE REVIEW

File: pyqueue_celery_processor/test/test_client.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import unittest
+from unittest.mock import patch
+from pyqueue_celery_processor.client import Client
```
Comment (mohit-fampay) [inline pyqueue_celery_processor/test/test_client.py]: Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

put a blank line between each group of imports.

File: pyqueue_celery_processor/test/test_consumer.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import unittest
+import queue
+from unittest.mock import Mock, patch
+from .tasks import sample_test_task
+from pyqueue_celery_processor.consumer import Consumer
```
Comment (mohit-fampay) [inline pyqueue_celery_processor/test/test_consumer.py]: Same, import-order-fix

File: pyqueue_celery_processor/test/__init__.py
Comment (mohit-fampay) [inline pyqueue_celery_processor/test/__init__.py:1]: Move tests under `tests` directory and not `test`

File: pyqueue_celery_processor/test/tasks.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from celery import Celery
+
+app = Celery('test')
+
+@app.task(name="sample test task")
+def sample_test_task(*args, **kwargs):
+    return {"args": args, "kwargs": kwargs}
```
Comment (mohit-fampay) [inline pyqueue_celery_processor/test/tasks.py]: Add newlines for each file.

File: setup.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 install_requires = [
     'prometheus-client==0.11.0',
+    'celery'
```
Comment (mohit-fampay) [inline setup.py]: since it's only been tested unser celery, `4.X.X` - Mention constraint here as well

**Review by mohit-fampay — CHANGES_REQUESTED** [review]

File: setup.py (hunk 4 above)
Comment (uzaxirr-fam) [reply setup.py]: locked version to 5.2.7 since this version is being used in both IB and westeros

File: setup.py [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 install_requires = [
     'prometheus-client==0.11.0',
+    'celery==5.2.7'
```
Comment (mohit-fampay) [inline setup.py]:
```suggestion
    'celery>=5.2.7'
```

**Review by mohit-fampay — CHANGES_REQUESTED** [review]

**Review by mohit-fampay — APPROVED** [review]

**Pull request merged**